### PR TITLE
Add Third Party and General Dependencies to README

### PR DESCRIPTION
**Pull request by talbs**
See https://github.com/edx/ux-pattern-library/blob/talbs/readme-add-dependencies/README.md for a preview of changes made.

- - -

##### Reviewers

* [x] Copy/Content - @marcotuts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +31,29 @@ The code in this repository is licensed the Apache 2.0 license unless otherwise
 noted. Please see the [LICENSE file](https://github.com/edx/ux-pattern-
 library/blob/master/LICENSE.txt) for details.
 
+## Dependencies
+
+Using the edX Pattern Library source code in a project current requires the following:
+
+* Your project use and can compile Sass into CSS using perferrably [LibSass](http://sass-lang.com/libsass). The edX Pattern Library currently does not provide pre-compiled CSS files.
+* You to locally install [Bower](http://bower.io/) - [see below](#development---ux-pattern-library-package) for more installation details.
+* Your project's Sass manually calls ``edx-pattern-library`` Sass partials in its main compile file.
```
Comment (marcotuts) [inline README.md]: I would suggest either altering the main sentence, or the first 3 bullet points to make the list context clearer. 

"project current(ly*) requires the following:" (of the 4 line items that follow, the last is clearest given this heading.) 

You might be able to get away with simply dropping "the following" (with some slight adjustment to the start of some of the bullets.

File: README.md [hunk 2]
```diff
@@ -30,6 +31,29 @@ The code in this repository is licensed the Apache 2.0 license unless otherwise
...
+Also, the Pattern Library currently relies on the following thrd party libraries:
+
+| Library  | Version |
+| ------------- | ------------- |
+| [Bourbon](https://github.com/thoughtbot/bourbon)  | 4.2.3  |
+| [Animate.css](https://github.com/daneden/animate.css)  | 3.3.0  |
+| [bi-app-sass](https://github.com/anasnakawa/bi-app-sass)  | latest  |
+| [breakpoint-sass](https://github.com/at-import/breakpoint)  | 2.6.1  |
+| [susy](https://github.com/ericam/susy)  | 2.2.5  |
+| [svg4everybody](https://github.com/jonathantneal/svg4everybody)  | 1.0.0  |
```
Comment (marcotuts) [inline README.md]: This is a helpful section! 

FWIW - an extra column with a 3-5 word summary per library might be helpful (but far from necessary!) (ex: Sass utilities and mixins [Bourbon], right to left Sass mixins [Bi-app-sass], etc.

File: README.md [hunk 3]
```diff
@@ -190,6 +214,8 @@ bower dependency and install it all at once.
 bower install edx-pattern-library --save
 ```
 
+**NOTE: If you have no initialized your project as a bower project, it's a generally a good thing to do to manage third party dependencies such as the ``edx-pattern-library`` . [Bower explains how to do that well with bower.json``](http://bower.io/docs/creating-packages/#bowerjson).
```
Comment (marcotuts) [inline README.md]: s/no/not

**Comment by marcotuts** [comment]
Small series of notes / nits here, but this looks great! :+1:

**Comment by talbs** [comment]
@marcotuts, thanks! Damn fine catches/recommendations - I'll sync those up and squash quickly.

**Comment by talbs** [comment]
@marcotuts, added those good suggestions - need another spin through?

**Comment by marcotuts** [comment]
looks great! :ship:

**Pull request merged**